Digitall.Persistance.EF/Configuration/EntityTypeConstants.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Digitall.Persistance.EF.Configuration;
+
+public static class EntityTypeConstants
```

Comment:
Should be in folder Constants

Reply:
I've added it to Configuration folder on root level to be as close to the configuration where these const will be most need it. For a single file I do not think it is need to be nested in the Constants folder.
![image](https://github.com/tpilitis/Digitall.Warehouse/assets/19794485/50da78dd-e721-43c9-aae0-d19bffb378de)
Looks good to me like it is. Over development if I see the need, I'll move it into a subfolder.